pymeasure/display/widgets/plot_widget.py

```
@@ -39,11 +40,12 @@ class PlotWidget(TabWidget, QtWidgets.QWidget):
     """ Extends :class:`PlotFrame<pymeasure.display.widgets.plot_frame.PlotFrame>`
     to allow different columns of the data to be dynamically chosen
     """
-
+    
     def __init__(self, name, columns, x_axis=None, y_axis=None, refresh_time=0.2,
-                 check_status=True, linewidth=1, parent=None):
+                 check_status=True, linewidth=1, selectors = [], parent=None):
```

Ruff (0.15.10)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)